src/operators/operators.h

```diff
@@ -325,9 +338,19 @@ class OperatorBase
     std::vector<std::vector<std::vector<std::vector<dealii::FullMatrix<real>>>>> gradient_mapping_shape_functions_face_flux_nodes; 
```

Comment:
Correct this quadruple `std::vector` following @dougshidong's comment

Reply:
If memory becomes an issue, might want to look into computing this on the fly in the `operators.cpp` file, something like `get_gradient_mapping_shape_functions_face_flux_nodes()` that takes the indices as arguments or whatever makes sense

Reply:
fixed.